### ISSUE TITLE
Implement the Logo Component

### ISSUE THREAD

**Issue by orama254** [reactdeveloperske/reactdevske-website#84]
### What feature are you adding?

Logo Component

### Requirements

- Create a component named `Logo` in the `components` folder that implements the screenshots below.
- It should accept a `size` as prop for the different sizes on the header and footer.
- The logo should rotate similar to how it does in a fresh create react app installation [as shown on this GIF](https://opensource.com/sites/default/files/uploads/react.gif).
- Obtain the logo from the Figma design. Use [THIS VIDEO](https://www.youtube.com/watch?v=T3rUlKf40M0) as a guide

**SCREENSHOTS**

[Figma Design](https://www.figma.com/file/TVwnaDhBGeVdnVKdf6H91C/React-developers-community-website?node-id=741%3A976)

### Acceptance Criteria

- [ ] The implementation should match the design.

**Comment by Ezekiel8807** [comment]
Assign this to me.... I would love to work on it

**Comment by orama254** [comment]
Hey @Ezekiel8807 

Go ahead, remember to go through our [contribution guidelines](https://github.com/reactdeveloperske/reactdevske-website/blob/main/CONTRIBUTING.md) and also evaluate the acceptance criteria before submitting a pull request.

Happy hacking!!!

**Comment by orama254** [comment]
@Ezekiel8807 

**ALSO NOTE:** This issue has some other dependencies, eg #77  & #83 How soon do you think you can submit a PR?

**Comment by orama254** [comment]
@yashsarnaik22 You can leave this issue to @Ezekiel8807 to contribute

**Comment by Ezekiel8807** [comment]
I'm on this already @yashsarnaik22

**Comment by Ezekiel8807** [comment]
If it's possible can you assign all other components related to the header to me

**Comment by yashsarnaik22** [comment]
okay @Ezekiel8807 you do the header part, I'll do the footer component

**Comment by Ezekiel8807** [comment]
please the link to the figma file where i can get the logo

**Comment by orama254** [comment]
My bad @Ezekiel8807 

It should be all good now, nice catch.

Let me know if there is anything else you would want assistance with.

**Comment by antosan** [comment]
> If it's possible can you assign all other components related to the header to me

@Ezekiel8807 Kindly do not combine this ticket and #77 in one PR. The reviewers need to have an easy time. Each ticket should be implemented in its own PR and tickets that are dependent on other tickets should be implemented first.

Kindly open a PR for the logo component first. Once that is accepted and merged, it will unblock #83 and it will also be possible to implement #77 in a separate PR.

**Comment by Ezekiel8807** [comment]
i have fixed the issue #84  in the PR #91 but it has a conflict that need to be resolved. i hope i can be given write access to resolve this conflict or maybe the reviewer can help me with that.
